frontend/package.json

```
@@ -8,6 +8,7 @@
     "dev:backend:dev": "NEXT_PUBLIC_BACKEND_TARGET=dev next dev",
     "build": "next build",
     "start": "next start",
+    "test": "vitest run --project unit",
```

Biome (2.5.6)

[error] The key test was already declared.

(lint/suspicious/noDuplicateObjectKeys)